libs/dao-data/src/utils/resolvers.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers } from 'ethers';
+import { QueryResult, resolvedMembership } from '..';
+import { ListMembershipsQuery } from '../subgraph/queries/members.generated';
+
+export const resolveMembershipList = (
```

Comment:
> Should this be a transformer because it is reformatting the data?

Oh yeah that is a better name. I cycled through hydrator and resolver. I'm going to make the name change.

![Autobots-Logo](https://user-images.githubusercontent.com/6923345/167150064-6846e9e1-59f5-4d60-85a0-91180f772de3.png)